external/st_audiofile/src/st_audiofile.c

```diff
@@ -566,15 +567,16 @@ uint64_t st_read_f32(st_audio_file* af, float* buffer, uint64_t count)
             if (!buf_i32) {
```

Comment:
For the float case a couple lines above, would we need to consider channel count too?

Reply:
I think the count value was multiplied in order to use for loop.
So we don't have to consider the channel for the float case.

Reply:
I think we needs it for drwav_f32_to_s16 function.